src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizerConfigurationValidator.cs

```diff
@@ -1,15 +1,27 @@
 ﻿using System;
-using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
 using Hl7.FhirPath;
+using Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging;
 
 namespace Fhir.Anonymizer.Core.AnonymizerConfigurations
 {
     public class AnonymizerConfigurationValidator
     {
+        private readonly ILogger _logger = AnonymizerLogging.CreateLogger<AnonymizerConfigurationValidator>();
+        
         public void Validate(AnonymizerConfiguration config)
-        {
+        {   
+            if (!string.IsNullOrEmpty(config.FhirVersion) && !string.Equals(Constants.supportedVersion, config.FhirVersion, StringComparison.InvariantCultureIgnoreCase))
+            {
+                throw new AnonymizerConfigurationErrorsException($"Configuration of fhirVersion {config.FhirVersion} is not supported. Expected fhirVersion: {Constants.supportedVersion}");
+            }
+            else
+            {
+                _logger.LogWarning($"Version is not specified in configuration file.");
```

Comment:
here seems would be a warning if the FhirVersion is equal to Constants.SupportedVersion